internal/db/schema/migrations/oss/postgres/47/01_roles.up.sql

```diff
@@ -0,0 +1,120 @@
+begin;
```

Comment:
[folder 47](https://github.com/hashicorp/boundary/tree/main/internal/db/schema/migrations/oss/postgres/47) has just been used by a go-kms-wrapping migration, so you'll need to bump this to 48

Reply:
Yep, just saw that --updated!